lib/fathead/perl_doc/Parsers/parse-faq.pl

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env perl
```

Comment:
@jbarrett No, no - you're idea was good. I've moved the aux parsers into the main file; we should be able to mix n' match parsers and ensure only a minimal amount of functionality is defined uniquely by each parser (easier to have shared routines).